docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
```

Comment:
If you plan to use ECS then be aware that `ecs-cli` compose do not support docker-compose file version 3. Also it does not support all the parameters, for e.g `depends_on` (`depends_on` is `links` in ecs) https://docs.aws.amazon.com/AmazonECS/latest/developerguide/cmd-ecs-cli-compose.html

Reply:
oh Thanks my docker master haha